b2sdk/file_version.py

```diff
@@ -488,23 +488,55 @@ def from_api_response(self, file_version_dict, force_action=None):
                "replicationStatus": "completed"
            }
 
+        or this:
+
+        .. code-block:: python
+
+            {
+                "content-length": 11, 
+                "content-type": "b2/x-auto", 
+                "x-bz-content-sha1": "2aae6c35c94fcfb415dbe95f408b9ce91ee846ed", 
+                "x-bz-upload-timestamp": 5000, 
+                "x-bz-file-id": "9999", 
+                "x-bz-file-name": "file"
+            }
+
         into a :py:class:`b2sdk.v2.FileVersion` object.
 
         """
         assert file_version_dict.get('action') is None or force_action is None, \
             'action was provided by both info_dict and function argument'
         action = file_version_dict.get('action') or force_action
-        file_name = file_version_dict['fileName']
-        id_ = file_version_dict['fileId']
-        if 'size' in file_version_dict:
-            size = file_version_dict['size']
-        elif 'contentLength' in file_version_dict:
-            size = file_version_dict['contentLength']
-        else:
+
+        if force_action is not None:
+            file_version_dict['action'] = force_action
+
+        file_name = file_version_dict.get('fileName', file_version_dict.get('x-bz-file-name'))
+        id_ = file_version_dict.get('fileId', file_version_dict.get('x-bz-file-id'))
+
+        if file_name is None:
```

Comment:
I'd prefer an assertion here and below:
```python
assert file_name, "no fileName ..."
```

Reply:
Are we ok with changing the semantics from raising ValueError to AssertionError? Probably we are, I just want to confirm whether I should do some additional checking/reviewing before I change it.